dev-setup/ubuntu/ubuntu-2004/SetupVMTest.txt

```diff
@@ -43,8 +43,8 @@
             distro==1.5.0 \
             execnet==1.8.0 \
             flake8==3.8.4 \
-            indy-plenum==1.13.0.dev1032 \
-            indy-node==1.13.0.dev1222 \
+            indy-plenum==1.13.0.dev14 \
+            indy-node==1.13.0.dev19 \
```

Comment:
This needs to be updated after the PR has been merged to the `dev` version of the GHA run